qcodes/instrument_drivers/AlazarTech/ATS9440.py

```diff
@@ -238,7 +238,7 @@ def __init__(self, name: str,
                                         'AD': 8,
                                         'BD': 9,
                                         'CD': 10,
-                                        'ABCD': 11})
```

Comment:
It looks like the mapping of D is probably also wrong then 7 should be changed to 8 and therefor AB, AD and CD are also wrong, could you check that?

Reply:
Yes, D was also wrong. I added another to commit to fix that.